doc/releases/pacific.rst

```diff
@@ -5,6 +5,185 @@ Pacific
 Pacific is the 16th stable release of Ceph.  It is named after the
 giant pacific octopus (Enteroctopus dofleini).
 
+v16.2.6 Pacific
+===============
+
+This is the sixth backport release in the Pacific series. We recommend all
+users update to this release.
+
+Notable Changes
+---------------
+
+* MGR: The pg_autoscaler has a new default 'scale-down' profile which provides more
+  performance from the start for new pools (for newly created clusters).
+  Existing clusters will retain the old behavior, now called the 'scale-up' profile.
+  For more details, see:
+
```

Comment:
I don't think there needs to be a blank space here.  Maybe just a newline because of word wrapping.